components/RecipeList.js

```diff
@@ -1,17 +1,23 @@
-import data from "@/db/data";
 import Image from "next/image";
 import styled from "styled-components";
 import Link from "next/link";
 
-export default function RecipeList() {
+export default function RecipeList({ recipes }) {
+  //Add empty entry for the Create Recipe Link/Symbol
+  const recipeList = [...recipes];
+
   return (
     <RecipeStyledUl>
-      {data.map((recipe) => (
-        <li key={recipe.name}>
+      {recipeList.map((recipe) => (
+        <li key={recipe.id}>
           <Link href={`/recipe/${recipe.id}`}>
             <StyledFigure>
               <Image
-                src={recipe.imageURL}
+                src={
+                  recipe.imageURL
+                    ? recipe.imageURL
+                    : "/recipe-images/generic-food-3.png"
+                }
```

Comment:
```suggestion
                src={
                  recipe.imageURL ?? "/recipe-images/generic-food-3.png"
                }
```

should do the same, if I am not wrong

Reply:
Unfortunately we were not able to use this because it broke everything. A empty string would be there and the image component would have a missing src=.